horace_core/sqw/file_io/class_helpers/is_sqw_struct.m

```diff
@@ -2,7 +2,11 @@
 % Check if input structure can be considered an sqw structure.
```

Comment:
Entire function could be:
```matlab
function is = is_sqw_struct(input)
    % Check if input structure can be considered an sqw structure.
    is = isa(input, 'binfile_v4_block_tester') || ...
         (isstruct(input) && all(isfield(input,{'main_header','header','detpar','data'})))
end
```
Which states intent directly

Reply:
I am hesitating. From one side, the form is shorter. From other, long logical operation sequences are difficult to read. 
Ok, probably one may use this in this particular case.